.agents/scripts/oauth-pool-lib/pool_ops_token_utils.py

```
@@ -0,0 +1,134 @@
+# google-validate: Validate token against Google API.
+# Env: ACCESS, HEALTH_URL
+# ---------------------------------------------------------------------------
+
+def cmd_google_validate() -> None:
+    from urllib.request import Request, urlopen
+    from urllib.error import HTTPError, URLError
+
+    token = os.environ['ACCESS']
+    url = os.environ['HEALTH_URL']
+    try:
+        req = Request(url, method='GET')
```

Ruff (0.15.9)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)